polling_stations/apps/data_importers/management/commands/import_sandwell.py

```diff
@@ -2,72 +2,50 @@
 
 
 class Command(BaseXpressDemocracyClubCsvImporter):
-    council_id = "E08000028"
-    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019sand.tsv"
-    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019sand.tsv"
-    elections = ["parl.2019-12-12"]
+    council_id = "SAW"
+    addresses_name = (
+        "2021-02-25T09:04:33.366338/Democracy Club Polling Place Lookup 6 May2021.tsv"
+    )
+    stations_name = (
+        "2021-02-25T09:04:33.366338/Democracy Club Polling Place Lookup 6 May2021.tsv"
+    )
+    elections = ["2021-05-06"]
     csv_delimiter = "\t"
-    allow_station_point_from_postcode = False
 
     def station_record_to_dict(self, record):
-        if record.polling_place_id == "17094":
-            # Hallam Street Methodist Church
+        if (
+            record.polling_place_id == "18846"
+        ):  # Hallam Street Methodist Church, Community Room, Lewisham Street, West Bromwich, B71 4HJ
             record = record._replace(polling_place_postcode="B71 4HG")
+
+        if (
+            record.polling_place_id == "18865"
+        ):  # St Philips Church Hall, Beeches Road, West Bromwich, B70 7PF
+            record = record._replace(polling_place_postcode="B70 6JA")
+
         return super().station_record_to_dict(record)
 
     def address_record_to_dict(self, record):
-        rec = super().address_record_to_dict(record)
         uprn = record.property_urn.strip().lstrip("0")
 
-        if record.post_code == "DY4 OQX":
-            rec["postcode"] = "DY40QX"
-        if record.post_code == "B7O 9LG":
-            rec["postcode"] = "B709LG"
-        if record.post_code == "DY4 ONT":
-            rec["postcode"] = "DY40NT"
-        if record.post_code == "DY4 OET":
-            rec["postcode"] = "DY40ET"
-        if record.post_code == "B70 OSH":
-            rec["postcode"] = "B700SH"
-        if record.post_code == "B68 OLH":
```

Comment:
Most of these fixes are still necessary.
The postcodes are invalid in the input data (they have `O` (capital 'O') instead of `0` (zero) - so it's worth fixing them, since we have these fixes here.